enhancements/installer/single-node-installation-bootstrap-in-place.md

```diff
@@ -171,8 +165,27 @@ logic, the modified script will:
    config, the control plane static pod manifests, the required
    kubernetes resources, and the bootstrap etcd database snapshot to
    create a new Ignition config for the host.
-3. Write the RHCOS image and the combined Ignition config to disk.
-4. Reboot the node.
+
+In order to successfully complete the installation, the user must tell
+`coreos-install` where to write the operating system image on the
+host's local storage. It may be necessary to provide other inputs to
+`coreos-install` at the same time (network addresses, etc.), and we
+have not yet completed the investigation and experimentation work
+required to design a supportable API for the complex details that may
+be needed. Therefore, to unblock progress on the bootstrap-in-place
+feature, we will temporarily treat installing the operating system as
+a manual step. Users running the OpenShift installer by hand will need
+to login to the host via `ssh` and run `coreos-install`, passing the
+appropriate arguments. Users relying on integration with Hive and the
+assisted installer for multi-cluster management will not need to
```

Comment:
I think we call it central cluster management but not 100% sure

Reply:
OK, I thought the term we've been using was "multi-cluster" but I'm happy to change it.